app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/EnterPasscodeFragment.kt

```diff
@@ -113,3 +113,9 @@ class EnterPasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>()
         }
     }
 }
+
+fun View.delayedShowKeyboardForView() {
```

Comment:
you could pass delay value as an argument with default value to this function

Reply:
I also moved it into a separate file: PasscodeUtils